src/web/extractors.rs

```diff
@@ -66,6 +66,24 @@ pub struct UidParam {
     uid: u64,
 }
 
+fn clean_entry(s: &str) -> Result<String, ApiError> {
+    println!("### GOT: {:?}", s);
```

Comment:
I assume these printlns are for debugging and will be deleted before the PR merges. If not, they should be converted to trace statements.